src/html5/node/data/element.rs

```diff
@@ -78,19 +78,19 @@ impl ElementAttributes {
     }
 
     /// Returns an iterator over the attribute map.
-    pub(crate) fn iter(&self) -> Iter<'_, String, String> {
+    pub(crate) fn iter(&self) -> impl Iterator<Item = (&String, &String)> {
```

Comment:
`impl Iterator` is generic and can be used almost anywhere, while `Iter` is a concrete type.